### ISSUE TITLE
Add a sidebar with the links to the ORCID(s) of the researcher(s) who created the portal

### ISSUE THREAD

**Issue by erinlynmclean** [NCEAS/metacatui#1739]
**Describe the feature you'd like**
Add a sidebar with the links to the ORCID(s) of the researcher(s) who created the portal

**Additional context**
Requested from the portals focus groups

**Comment by laurenwalker** [comment]
@erinlynmclean Looking at the focus group doc, it looks like this feature is specifically about adding a filter to the query builder for easily filtering the portal creator's datasets? Let me know if I'm misunderstanding that.

**Comment by erinlynmclean** [comment]
No, it was more about transparency behind who created the portal. like, when i read a news article, I can do some research on the author if i'd like to see what other things that person has written...same concept.

**Comment by laijasmine** [comment]
maybe something like the member page here: https://arcticdata.io/catalog/portals/DBO/Members?

**Comment by erinlynmclean** [comment]
yeah, but something automatic so that people don't have to create that freeform page and then maintain it.

**Comment by laurenwalker** [comment]
Got it, thanks!
